src/Mechanisms/HandleRequests/HandleRequests.php

```diff
@@ -104,8 +105,21 @@ function handleUpdate()
             'assets' => SupportScriptsAndAssets::getAssets(),
         ];
 
-        $finish = trigger('response', $responsePayload);
+        $response = $finish($responsePayload);
+
+        /**
+         * Headers have already been sent by stream
+         * we need to send the output without additional headers
+         **/
+        if (headers_sent()) {
+            return new class(json_encode($response)) extends Response {
```

Comment:
If we return this new response here, none of the actual `$response` get sent though right? I wonder if that will break anything for Livewire as no component state, etc will be send. 🤔